--- AppDelegate.m	2020-11-14 23:55:37.000000000 -0500
+++ AppDelegate.m	2020-11-14 23:39:56.000000000 -0500
@@ -13,7 +13,7 @@
 #import "RNQuickActionManager.h"
 #import <UserNotifications/UserNotifications.h>
 #import <RNCPushNotificationIOS.h>
-#ifdef FB_SONARKIT_ENABLED
+#if !TARGET_OS_MACCATALYST
 #if DEBUG
 #import <FlipperKit/FlipperClient.h>
 #import <FlipperKitLayoutPlugin/FlipperKitLayoutPlugin.h>
@@ -38,10 +38,11 @@
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-#ifdef FB_SONARKIT_ENABLED
+#if DEBUG
+  #if !TARGET_OS_MACCATALYST
   InitializeFlipper(application);
 #endif
-
+#endif
   RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge
                                                    moduleName:@"BlueWallet"
-
