--- Podfile	2020-11-15 00:35:20.000000000 -0500
+++ Podfile	2020-11-15 00:36:56.000000000 -0500
@@ -8,24 +8,21 @@
 
   use_react_native!(:path => config["reactNativePath"])
 
+  permissions_path = '../node_modules/react-native-permissions/ios'
+  pod 'Permission-LocationWhenInUse', :path => "#{permissions_path}/LocationWhenInUse.podspec"
   # Enables Flipper.
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
   post_install do |installer|
-    flipper_post_install(installer)
+    installer.pods_project.targets.each do |target|
+        target.build_configurations.each do |config|
+          if ['React-Core-AccessibilityResources'].include? target.name 
+            config.build_settings['CODE_SIGN_STYLE'] = "Manual"
+            config.build_settings['CODE_SIGN_IDENTITY'] = "Apple Distribution"
+            config.build_settings['DEVELOPMENT_TEAM'] = "A7W54YZ4WU"
+          end
+        end
+    end
   end
-end
-
-target 'MarketWidgetExtension' do
-  pod 'SwiftSocket', :git => 'https://github.com/swiftsocket/SwiftSocket.git', :branch => 'master'
-end
-
-target 'WalletInformationAndMarketWidgetExtension' do
-  pod 'SwiftSocket', :git => 'https://github.com/swiftsocket/SwiftSocket.git', :branch => 'master'
-end
-
-target 'WalletInformationWidgetExtension' do
-  pod 'SwiftSocket', :git => 'https://github.com/swiftsocket/SwiftSocket.git', :branch => 'master'
-end
+end
\ No newline at end of file
