--- Podfile	2020-11-19 00:16:58.000000000 -0500
+++ p	2020-11-19 00:16:46.000000000 -0500
@@ -12,9 +12,7 @@
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
   post_install do |installer|
-    flipper_post_install(installer)
       installer.pods_project.targets.each do |target|
           target.build_configurations.each do |config|
             if ['React-Core-AccessibilityResources'].include? target.name 
@@ -28,15 +26,3 @@
 
   
 end
-
-target 'MarketWidgetExtension' do
-  pod 'SwiftSocket', :git => 'https://github.com/swiftsocket/SwiftSocket.git', :branch => 'master'
-end
-
-target 'WalletInformationAndMarketWidgetExtension' do
-  pod 'SwiftSocket', :git => 'https://github.com/swiftsocket/SwiftSocket.git', :branch => 'master'
-end
-
-target 'WalletInformationWidgetExtension' do
-  pod 'SwiftSocket', :git => 'https://github.com/swiftsocket/SwiftSocket.git', :branch => 'master'
-end
