diff --git a/node_modules/@lodev09/react-native-true-sheet/android/src/main/java/com/lodev09/truesheet/TrueSheetViewManager.kt b/node_modules/@lodev09/react-native-true-sheet/android/src/main/java/com/lodev09/truesheet/TrueSheetViewManager.kt
index 2d36d8b..86f6343 100644
--- a/node_modules/@lodev09/react-native-true-sheet/android/src/main/java/com/lodev09/truesheet/TrueSheetViewManager.kt
+++ b/node_modules/@lodev09/react-native-true-sheet/android/src/main/java/com/lodev09/truesheet/TrueSheetViewManager.kt
@@ -95,9 +95,10 @@ class TrueSheetViewManager : ViewGroupManager<TrueSheetView>() {
   }
 
   @ReactProp(name = "background")
   fun setBackground(view: TrueSheetView, colorName: Double) {
-    val color = runCatching { ColorPropConverter.getColor(colorName, view.context) }.getOrDefault(Color.WHITE)
+    val color = runCatching { ColorPropConverter.getColor(colorName, view.context) }
+      .getOrDefault(Color.WHITE) ?: Color.WHITE
     view.setBackground(color)
   }
 
   @ReactProp(name = "sizes")
*** End Patch