diff --git a/node_modules/metro-minify-uglify/src/minifier.js b/node_modules/metro-minify-uglify/src/minifier.js
index b703ee4..fadc077 100644
--- a/node_modules/metro-minify-uglify/src/minifier.js
+++ b/node_modules/metro-minify-uglify/src/minifier.js
@@ -67,9 +67,7 @@ function minify(_ref) {
     config = _ref.config;
 
   const options = _objectSpread({}, config, {
-    mangle: _objectSpread({}, config.mangle, {
-      reserved
-    }),
+    mangle: false, // !!!!!!!!!!!!!!!!!!!!!!!!
     sourceMap: _objectSpread({}, config.sourceMap, {
       content: map
     })
