diff --git a/node_modules/react-native-capture-protection/android/src/main/java/com/captureprotection/CaptureProtectionModule.kt b/node_modules/react-native-capture-protection/android/src/main/java/com/captureprotection/CaptureProtectionModule.kt
index 40bebef..47d5e7c 100644
--- a/node_modules/react-native-capture-protection/android/src/main/java/com/captureprotection/CaptureProtectionModule.kt
+++ b/node_modules/react-native-capture-protection/android/src/main/java/com/captureprotection/CaptureProtectionModule.kt
@@ -22,7 +22,7 @@ class CaptureProtectionModule(private val reactContext: ReactApplicationContext)
   }
 
   @ReactMethod
   fun hasListener(promise: Promise) {
-    currentActivity?.runOnUiThread {
+    reactContext.currentActivity?.runOnUiThread {
       try {
         val params = super.hasScreenCaptureListener()
         promise.resolve(params)
@@ -33,7 +33,7 @@ class CaptureProtectionModule(private val reactContext: ReactApplicationContext)
   }
 
   @ReactMethod
   fun isScreenRecording(promise: Promise) {
-    currentActivity?.runOnUiThread {
+    reactContext.currentActivity?.runOnUiThread {
       try {
         promise.resolve(super.screens.size > 1)
@@ -44,7 +44,7 @@ class CaptureProtectionModule(private val reactContext: ReactApplicationContext)
   }
 
   @ReactMethod
   fun prevent(promise: Promise) {
-    currentActivity?.runOnUiThread {
+    reactContext.currentActivity?.runOnUiThread {
       try {
         val currentActivity = ActivityUtils.getReactCurrentActivity(reactContext)
@@ -60,7 +60,7 @@ class CaptureProtectionModule(private val reactContext: ReactApplicationContext)
   }
 
   @ReactMethod
   fun allow(promise: Promise) {
-    currentActivity?.runOnUiThread {
+    reactContext.currentActivity?.runOnUiThread {
       try {
         val currentActivity = ActivityUtils.getReactCurrentActivity(reactContext)
@@ -75,7 +75,7 @@ class CaptureProtectionModule(private val reactContext: ReactApplicationContext)
   }
 
   @ReactMethod
   fun protectionStatus(promise: Promise) {
-    currentActivity?.runOnUiThread {
+    reactContext.currentActivity?.runOnUiThread {
       try {
         val flags = ActivityUtils.isSecureFlag(reactContext)
