diff --git a/node_modules/react-native-worklets-core/android/build.gradle b/node_modules/react-native-worklets-core/android/build.gradle
index b6af1c3..46ee7cc 100644
--- a/node_modules/react-native-worklets-core/android/build.gradle
+++ b/node_modules/react-native-worklets-core/android/build.gradle
@@ -145,6 +145,8 @@ android {
       "**/libfolly_runtime.so",
       "**/libglog.so",
       "**/libhermes.so",
+      "**/libhermestooling.so",
+      "**/libhermesvm.so",
       "**/libhermes-executor-debug.so",
       "**/libhermes_executor.so",
       "**/libreactnative.so",
