diff --git a/node_modules/react-native-screens/ios/RNSBarButtonItem.mm b/node_modules/react-native-screens/ios/RNSBarButtonItem.mm
index 76a39d4..5b7f2d9 100644
--- a/node_modules/react-native-screens/ios/RNSBarButtonItem.mm
+++ b/node_modules/react-native-screens/ios/RNSBarButtonItem.mm
@@ -81,6 +81,7 @@ - (instancetype)initWithConfig:(NSDictionary<NSString *, id> *)dict
     NSString *identifier = dict[@"identifier"];
     if (identifier != nil) {
       self.identifier = identifier;
+      self.accessibilityIdentifier = identifier;
     }
     NSDictionary *badgeConfig = dict[@"badge"];
     if (badgeConfig != nil) {
@@ -208,9 +209,11 @@ + (UIAction *)createActionItemFromConfig:(NSDictionary *)dict
 {
   NSString *menuId = dict[@"menuId"];
 
+  NSString *identifier = dict[@"identifier"];
+
   UIAction *actionElement = [UIAction actionWithTitle:dict[@"title"]
-                                                image:nil
-                                           identifier:nil
+                                               image:nil
+                                           identifier:identifier
                                               handler:^(__kindof UIAction *_Nonnull a) {
                                                 menuAction(menuId);
                                               }];
