diff --git a/node_modules/react-native-screens/ios/RNSBarButtonItem.mm b/node_modules/react-native-screens/ios/RNSBarButtonItem.mm
index 0eb1f09..6d9d437 100644
--- a/node_modules/react-native-screens/ios/RNSBarButtonItem.mm
+++ b/node_modules/react-native-screens/ios/RNSBarButtonItem.mm
@@ -81,6 +81,7 @@ static UIMenuOptions RNSMakeUIMenuOptionsFromConfig(NSDictionary *config);
     NSString *identifier = dict[@"identifier"];
     if (identifier != nil) {
       self.identifier = identifier;
+      self.accessibilityIdentifier = identifier;
     }
     NSDictionary *badgeConfig = dict[@"badge"];
     if (badgeConfig != nil) {
@@ -201,9 +202,11 @@ static UIMenuOptions RNSMakeUIMenuOptionsFromConfig(NSDictionary *config);
 {
   NSString *menuId = dict[@"menuId"];
 
+  NSString *identifier = dict[@"identifier"];
+
   UIAction *actionElement = [UIAction actionWithTitle:dict[@"title"]
-                                                image:nil
-                                           identifier:nil
+                                               image:nil
+                                           identifier:identifier
                                               handler:^(__kindof UIAction *_Nonnull a) {
                                                 menuAction(menuId);
                                               }];
