diff --git a/node_modules/react-native-blue-crypto/android/build.gradle b/node_modules/react-native-blue-crypto/android/build.gradle
index 3711bba..4f9813a 100644
--- a/node_modules/react-native-blue-crypto/android/build.gradle
+++ b/node_modules/react-native-blue-crypto/android/build.gradle
@@ -4,12 +4,11 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
-    // noinspection DifferentKotlinGradleVersion
+    classpath 'com.android.tools.build:gradle:8.7.2'
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
   }
 }
@@ -45,14 +44,13 @@ android {
     disable 'GradleCompatible'
   }
   compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
+    sourceCompatibility JavaVersion.VERSION_17
+    targetCompatibility JavaVersion.VERSION_17
   }
 }
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
